components/chat_widget/src/components/ocs-chat/ocs-chat.tsx

```diff
@@ -705,6 +725,32 @@ export class OcsChat {
     }
   }
 
+  private getOrGenerateUserId(): string {
+    if (this.userId) {
+      return this.userId;
+    }
+
+    if (this.generatedUserId) {
+      return this.generatedUserId;
+    }
+
+    const storageKey = `ocs-user-id-${this.chatbotId}`;
```

Comment:
This should be consistent across multiple bots.
```suggestion
    const storageKey = `ocs-user-id`;
```